message_definitions/v1.0/ras_a.xml

```diff
@@ -0,0 +1,124 @@
+        Detection time happens once for the same UID, while time of update changes when a specific metadata of that POI gets changed (e.g. position).
+        The time of update should be changed on the sending system, based on the determined data in regards to that specific POI.
+        So, POIs that are received again should be updated if the time of update has changed.
+        Note: The sending system should repeat the current POIs at a fixed default rate of 2Hz to keep the protocol stateless). The fixed rate though can be set by the receiver using MAV_CMD_SET_MESSAGE_INTERVAL, which is advised for POIs that are not static or for which the state updates are high - decision on the rates of some specific POIs is at the implementers consideration.
+      </description>
+      <field type="uint32_t" name="time_boot_ms" units="ms">Timestamp (time since system boot).</field>
+      <field type="uint64_t" name="time_utc_detected" units="ms" invalid="0">Timestamp (time since UNIX epoch) of the POI detection, in UTC. 0 for unknown.</field>
+      <field type="uint64_t" name="time_utc_updated" units="ms" invalid="0">Timestamp (time since UNIX epoch) of the last POI update, in UTC. 0 for unknown.</field>
+      <field type="uint8_t" name="confidence_overall" units="%" invalid="UINT8_MAX">Generic confidence level. Can be used for an implementation specific confidence level. 0..100, UINT8_MAX when unknown.</field>
+      <field type="uint8_t" name="confidence_detection" units="%" invalid="UINT8_MAX">Confidence level of the POI detection. 0..100, UINT8_MAX when unknown.</field>
+      <field type="uint8_t" name="confidence_classification" units="%" invalid="UINT8_MAX">Confidence level of the POI classification. 0..100, UINT8_MAX when unknown.</field>
+      <field type="uint8_t" name="confidence_localization" units="%" invalid="UINT8_MAX">Confidence level of the POI localization. 0..100, UINT8_MAX when unknown.</field>
+      <field type="uint16_t" name="ttl" units="s">Time to live: If this time has elapsed, the POI should be deleted on the receiver side. A value of 0 should indicate no timeout.</field>
+      <field type="uint8_t" name="flags" display="bitmask">Bitmask for POI state. Bit 1: POI is in focus on camera, Bit 8: POI has been cleared and should be deleted.</field>
+      <field type="uint64_t" name="uid" invalid="0">Unique ID for a given POI. Updates to a POIs information should use the same uid. 0 means unknown.</field>
+      <field type="int32_t" name="latitude" units="degE7" invalid="INT32_MAX">Latitude (WGS84) of the POI. If unknown: INT32_MAX (both Lat/Lon).</field>
+      <field type="int32_t" name="longitude" units="degE7" invalid="INT32_MAX">Longitude (WGS84) of the POI. If unknown: INT32_MAX (both Lat/Lon).</field>
+      <field type="int32_t" name="alt_amsl" units="mm" invalid="INT32_MAX">Altitude AMSL of the POI. Positive for up. INT32_MAX if unknown.</field>
```

Comment:
It is more common to use a `float` and specify the value in m. This is fine though. I presume you need the mm resolution across a broad range.

Is there a need for POI altitude relative to terrain?

Reply:
Actually this should be a float because you can be bellow MSL. So I will update it.

> Is there a need for POI altitude relative to terrain?

Probably. I will add that one.